### PR TITLE
docs: apply improvement suggestions

### PR DESCRIPTION
**Pull request by JuroUhlar**
Combination of Louise's suggestions and my own: 

https://github.com/orgs/fingerprintjs/projects/7/views/5?filterQuery=assignee%3AJuroUhlar&pane=issue&itemId=22881147

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -15,54 +15,88 @@
...
+
+### Fingerprint Pro
+
+[Fingerprint Pro](https://fingerprint.com/github/) offers highly accurate browser fingerprinting technology (99.5%) and is used by over 8,000 websites. It can verify legitimate users and block fraudsters even if they use VPNs, browse in incognito  mode, clear their cookies, or use other methods to hide their identity. Fingerprint Pro helps companies prevent fraud, improve user experiences, and better understand their traffic.
+
+Fingerprint Pro builds on the FingerprintJS browser fingerprint library to provide up to 99.5% accurate device identification. Key capabilities include:
+
+- Cookie and local storage management - Leverages cryptographically signed cookies and local storage to reliably associate visitor identifier values with fingerprintable attributes.
+- Server-side analysis and machine learning - Provides a holistic view of all attributes and layers to generate the most stable and accurate visitor ID.
+- Storage and deduplication - Collects multiple attributes to do fuzzy matching and handle browser and OS upgrades
```
Comment (makma) [inline readme.md:34]:
```suggestion
- Storage and deduplication - Collects multiple attributes to do fuzzy matching and handle browser and OS upgrades.
```

**Review by makma — APPROVED** [review]
Thank you @JuroUhlar, great improvements!

File: readme.md (hunk 1 above)
Comment (JuroUhlar) [reply readme.md]: Thanks! Now I need the approval again  🙂

**Comment by ilfa** [comment]
We can also mention `flutter` and `react-native` integrations.

**Comment by JuroUhlar** [comment]
> We can also mention `flutter` and `react-native` integrations.

@ilfa Good point, thanks, done!

**Review by ilfa — APPROVED** [review]

**Review by makma — APPROVED** [review]

**Pull request merged**